src/model/dataTypes/ComplexDataType.ts

```diff
@@ -2,14 +2,50 @@ import { normalizeTypeName } from "../../utils";
 import IDataType from "./IDataType";
 import { primitiveTypeCheck } from "./primitiveDataTypes";
 
+// This cache is a map of namespace strings, whose objects are a map of types under that namespace
+// E.g.: { "FHIR": { "Account": { /* Account obj }, "Activity": { /* Activity Obj */ } }
+interface ICachedNamespace {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5648116)